homeassistant/helpers/selector.py

```diff
@@ -160,6 +160,18 @@ class BaseSelectorConfig(TypedDict, total=False):
 )
 
 
+LEGACY_ENTITY_SELECTOR_CONFIG_SCHEMA = vol.Schema(
```

Comment:
Please add some comment explaining what "LEGACY" refers to, same for the legacy device selector schema.